schema/annotation-source.yaml

```diff
@@ -67,12 +68,15 @@ $defs:
         const: Contribution
         default: Contribution
         description: MUST be "Contribution".
-      agent:
+      contribution_made_by:
```

Comment:
As noted, VA Core model tends toward attribute names that describe the relationship  between its value and the entity it hangs from. 
Other names we might consider: 'contributor'?  'contributing_agent'?

Reply:
so when `subject`, `method` and other attributes that hang from Statement or Proposition we don't qualify them with some relationship to the entity they hang from? I'm not sure I see the consistency here?  
That said, if there is some rationale that is being applied consistently here and I'm not seeing it, then I would prefer `contributor` which seems the simplest and clearest.  

(Maybe the rationale is that we don't use the "type" value for the attribute name?)

Reply:
> so when subject, method and other attributes that hang from Statement or Proposition we don't qualify them with some relationship to the entity they hang from? I'm not sure I see the consistency here?

I think I am making a much simpler point that the names VA chooses for attributes/properties in the VA core model tend to express _how the value of the attribute is related to the entity it hangs from_, as opposed to being named after _the type of data or object it holds_.  For example, we choose the name `Contribution.contribution_made_by` for an attribute that captures the agent who made a particular contribution, rather than `Contribution.agent` - because it is more descriptive of how the value of this attribute relates to a Contribution object. The attribute name 'agent' simply reflects what type of entity goes into the field - but does not indicate how this agent relates to the Contribution (this is obvious in this case, but isn't always so). You'll find lots of other examples of this naming approach in the VA model.

As for the name we chose for this specific property in the schema, I could work with `contributor` if you prefer it.  Esp. since it pairs nicely with `contributor_role` (if we end up keeping this name).

Reply:
> (Maybe the rationale is that we don't use the "type" value for the attribute name?)

I don't quite follow - but suspect this may be revealing an important disconnect in how you and I are thinking - so lets clarify when we talk.